### ISSUE TITLE
Assessment Profile: assessmentDeliveryType and learningDeliveryType

### ISSUE THREAD

**Issue by siuc-nate** [CredentialTransparencyInitiative/vocabularies#67]
Currently the vocabulary includes both "assessmentDeliveryType" and "learningDeliveryType".  AssessmentDeliveryType also indicates in its usage note that its values are to be drawn from "Assessment Modality", which is a third property, assessmentModalityType (which oddly does not have this usage note).

My understanding is that learningDeliveryType is the correct property/vocabulary, and was intended to replace assessmentDeliveryType. Is this correct?

**Comment by stuartasutton** [comment]
@siuc-nate & @jkitchensSIUC , not sure I understand. The vocabularies spreadsheet [1] defines both ***ctdl:AssessmentDelivery*** and ***ctdl:LearningDelivery*** vocabularies. Also, per your comment, Nate, the ctdl:assessmentDeliveryType property [2] has no such usage note (where are you seeing it?). So, we have two properties and two companion vocabularies. 

@siuc-nate, are you saying:

1. ***Keep*** ctdl:learningDeliveryType and the companion vocabulary ctdl:LearningDelivery; and
2. ***Delete*** ctdl:assessmentDeliveryType and the companion vocabulary ctdl:LearningDelivery?

[1] https://docs.google.com/spreadsheets/d/1iRj0vYCV3ZYETmQMLPNB2plmrxx50eFHWCBeCERZx14/edit#gid=0
[2] http://www.credreg.net/page/propertiesviewer#assessmentDeliveryType

**Comment by siuc-nate** [comment]
I had moved the usage note during yesterday's changes.  Forgot about that one, sorry. 

As for the rest, since we're using learningDeliveryType with learningOpportunity, i think it makes sense to keep it and drop assessmentDelivery.  

The AssessmentDelivery vocabulary includes these three terms:
- inPerson
- online
- proctored

LearningDelivery has "online" already. It also has "faceToFace", which (comparing the definitions) is equivalent to "inPerson".  If we add "proctored" to LearningDelivery, and decide whether or not "inPerson" is better than "faceToFace" as a term, then LearningDelivery will render AssessmentDelivery unnecessary.

In other words, I think we should merge the vocabularies since one nearly encompasses the other, and drop the smaller of the two vocabularies along with the property that points to it.

**Comment by stuartasutton** [comment]
I've no problem with this approach, Nate (i.e. combining the concepts into one vocabulary). However, we've got in front of the assessment folk the AssessmentDelivery vocabulary for review. Do we need to be cautious? @jkitchensSIUC, what do you think (don't see Bob's Github ID).

**Comment by siuc-nate** [comment]
This is covered in #84
